third-party/externals/autoconf-archive.cmake

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2019 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+
+set(name autoconf-archive)
+set(source_dir ${CMAKE_CURRENT_BINARY_DIR}/${name}/source)
+ExternalProject_Add(
+    ${name}
+    URL https://ftp.gnu.org/gnu/autoconf-archive/autoconf-archive-2019.01.06.tar.xz
```

Comment:
It will be decompressed automatically?

Reply:
Yeah, downloading, verifying, decompression and building are all builtin function of `ExternalProject_Add`.